packages/fast-jss-manager-react/src/jss-manager.spec.tsx

```diff
@@ -60,201 +72,127 @@ const staticAndDynamicStyles: ComponentStyles<any, any> = {
-        expect(objectStylesheetComponent.state("styleSheet").attached).toBe(true);
-
-        expect(functionStylesheetComponent.state("styleSheet")).toBeInstanceOf(
-            StyleSheet
-        );
-        expect(functionStylesheetComponent.state("styleSheet").attached).toBe(true);
+        expect(rendered.find("SimpleComponent").prop("jssStyleSheet")).toBeUndefined();
     });
 
-    test("should update an object stylesheet when the design-system changes", (): void => {
-        const objectStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                designSystem={{ color: "blue" }}
-                styles={stylesheet}
-                render={renderChild}
-            />
-        );
+    test("should not pass the managedClasses prop through to managed component", (): void => {
+        const managedClasses: any = { foo: "foo" };
+        const rendered: any = mount(<NoPropsManager managedClasses={managedClasses} />);
 
-        const functionStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                designSystem={{ color: "blue" }}
-                styles={functionStyleSheet}
-                render={renderChild}
-            />
+        expect(rendered.find("SimpleComponent").prop("managedClasses")).not.toEqual(
+            managedClasses
         );
+    });
 
-        const mock: any = jest.fn();
-
-        objectStylesheetComponent.state("styleSheet").update = mock;
-        objectStylesheetComponent.setProps({ designSystem: testDesignSystem });
+    test("should provide an empty object to the managedClasses prop if no styles are provided", (): void => {
+        const rendered: any = mount(<NoPropsManager />);
 
-        expect(mock.mock.calls).toHaveLength(1);
-        expect(mock.mock.calls[0][0]).toEqual(testDesignSystem);
+        expect(rendered.find("SimpleComponent").prop("managedClasses")).toEqual({});
+    });
 
-        const functionSheet: any = functionStylesheetComponent.state("styleSheet");
-        functionStylesheetComponent.setProps({
-            designSystem: testDesignSystem,
-        });
+    test("should have a default context if no context is provided", (): void => {
+        const rendered: any = mount(<NoPropsManager managedClasses={{ foo: "foo" }} />);
 
-        // Function stylesheets must be completely re-generated when the design-system changes,
-        // so check identity
-        expect(functionStylesheetComponent.state("styleSheet")).not.toBe(functionSheet);
+        expect(rendered.instance().context).toEqual({});
     });
 
-    test("should remove stylesheets when unmounting", (): void => {
-        const objectStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                designSystem={{ color: "red" }}
-                styles={stylesheet}
-                render={renderChild}
-            />
+    test("the default context should share identity between component instances", (): void => {
```

Comment:
"should have a default context that shares identity between component instances"